AsyncDisplayKit/Private/ASTranslationRange.h

```diff
@@ -0,0 +1,47 @@
+//
+//  ASTranslationRange.h
+//  AsyncDisplayKit
+//
+//  Created by Levi McCallum on 4/4/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+/**
+ * A translation range simulates a contiguous range, but allows offsets to be inserted and removed to translate an index
+ * within the contiguous range to the actual index that
```

Comment:
Description cut short.

Love this class.  I don't yet fully understand a simple way to explain what it does, and we might want to consider different names, but just upgrading the description could be enough.